### ISSUE TITLE
`sv:runReport:report` fails due to malformed `.wf_sv_clinvar.vcf` header error

### ISSUE THREAD

**Issue by BeneKenobi** [epi2me-labs/wf-human-variation#78]
### Operating System

CentOS 7

### Workflow Version

v1.7.1

### Workflow Execution

Command line

### CLI command run

nextflow run epi2me-labs/wf-human-variation
		 -r v1.7.1
		 -w /hpc/scratch/project/humangenetik/lonGER/pipeline_output/workspace
		 -bg
		 -profile singularity
		 -name research_wf-human-variation_XXX_20230828093246
		 --sv
		 --snp
		 --cnv
		 --str
		 --methyl
		 --sample_name XXX
		 --sex female
		 --fast5_dir /hpc/scratch/project/humangenetik/lonGER/raw_data/XXX
		 --dorado_ext pod5
		 --out_dir /hpc/scratch/project/humangenetik/lonGER/pipeline_output/XXX
		 --ref /hpc/scratch/project/humangenetik/references/GRCh38.fa
		 --basecaller_cfg dna_r10.4.1_e8.2_400bps_sup@v4.2.0
		 --remora_cfg dna_r10.4.1_e8.2_400bps_sup@v4.2.0_5mCG_5hmCG@v2

### Workflow Execution - CLI Execution Profile

None

### What happened?

The workflow fails, no report is generated.

### Relevant log output

```shell
[05:28:30 - workflow_glue] Starting entrypoint.
[E::bcf_hdr_parse] Could not parse the header, sample line not found
Traceback (most recent call last):
  File "pysam/libcbcf.pyx", line 4355, in pysam.libcbcf.VariantFile.open
  File "pysam/libcbcf.pyx", line 2210, in pysam.libcbcf.makeVariantHeader
ValueError: cannot create VariantHeader

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/hpc/scratch/project/humangenetik/applications/nextflow_home/assets/epi2me-labs/wf-human-variation/bin/workflow-glue", line 7, in <module>
    cli()
  File "/hpc/scratch/project/humangenetik/applications/nextflow_home/assets/epi2me-labs/wf-human-variation/bin/workflow_glue/__init__.py", line 72, in cli
    args.func(args)
  File "/hpc/scratch/project/humangenetik/applications/nextflow_home/assets/epi2me-labs/wf-human-variation/bin/workflow_glue/report_sv.py", line 227, in main
    clinvar_for_report = load_clinvar_vcf(sample_vcf)
  File "/home/epi2melabs/conda/lib/python3.8/site-packages/ezcharts/components/clinvar.py", line 254, in load_clinvar_vcf
    dfs = [load_vcf(vcf_fn, benign=benign, all_sites=all_sites)]
  File "/home/epi2melabs/conda/lib/python3.8/site-packages/ezcharts/components/clinvar.py", line 70, in load_vcf
    vcf_file = VariantFile(vcf_fn)
  File "pysam/libcbcf.pyx", line 4117, in pysam.libcbcf.VariantFile.__init__
  File "pysam/libcbcf.pyx", line 4357, in pysam.libcbcf.VariantFile.open
ValueError: file `b'XXX.wf_sv_clinvar.vcf'` does not have valid header (mode=`b'r'`) - is it VCF/BCF format?
```

**Comment by RenzoTale88** [comment]
Hi @BeneKenobi it seems the VCF file seems to lack the sample line. 
Do you mind sharing the header, or check if the file has an header in the first place?
```
grep '#' XXX.wf_sv_clinvar.vcf
```

**Comment by BeneKenobi** [comment]
No, it doesn't.

[XXX.wf_sv_clinvar.vcf.zip](https://github.com/epi2me-labs/wf-human-variation/files/12462017/XXX.wf_sv_clinvar.vcf.zip)

**Comment by RenzoTale88** [comment]
@BeneKenobi this should now be fixed in v1.7.2. Could you please try and see if that works?

**Comment by BeneKenobi** [comment]
Yes it worked! Finally I got the whole pipeline working for the first time. 🎉
